catalog.html

```diff
@@ -0,0 +1,229 @@
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Каталог</title>
+  </head>
+  <body>
+    <header class="page-header">
+
+      <a href="#"><img class="logo" src="#" alt="Логотип интернет-магазина девайсов»."></a>
+
+      <div>
+
+       <p class="navigation-item">
+          <a class="navigation-link" href="#">Войти</a></p>
+      </div>
+
+      <ul class="navigation-list navigation-user">
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">Сравнить</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">Корзина</a>
+        </li>
+      </ul>
+
+      <nav class="navigation">
+        <p>Каталог товаров</p>
```

Comment:
```suggestion
        <a href="catalog.html">Каталог товаров</a>
```
Пока можно и так, но удобнее, пока нет выпадающего меню, сделать ссылкой на вторую страницу.